ingestion/src/metadata/ingestion/source/database/bigquery/profiler/profiler.py

```diff
@@ -0,0 +1,26 @@
+from typing import List, Type
+
+from metadata.generated.schema.entity.data.table import SystemProfile
+from metadata.ingestion.source.database.bigquery.profiler.system import (
+    BigQuerySystemMetricsComputer,
+)
+from metadata.profiler.interface.sqlalchemy.bigquery.profiler_interface import (
+    BigQueryProfilerInterface,
+)
+from metadata.profiler.metrics.system.system import System
+from metadata.profiler.processor.runner import QueryRunner
+
+
+class BigQueryProfiler(BigQueryProfilerInterface):
+    system_metrics_computer_class = BigQuerySystemMetricsComputer
```

Comment:
Should the metric computation definition be coupled with the profiler class? I believe right now this is somewhat decouple where the metrics computation logic lives in the metrics module and the metric module will either use the base definition and use the one related to the source -- but there is no metric injection directly into the profiler.